src/Connect/index.tsx

```diff
@@ -86,6 +87,15 @@ const ConnectScreen: FunctionComponent = () => {
     listItems.push(reportAnIssue)
   }
 
+  if (displayCallbackForm) {
+    const callbackForm: ConnectListItem = {
```

Comment:
I like this pattern, where you only create the `callbackForm` if `displayCallbackForm` is true. Is this better in some way than creating `callbackForm` outside the if block and then only pushing it to the list items array if `displayCallbackForm` is true? I'm curious as to why you would do it this way vs. the other.

Reply:
It was mostly to keep the usage close to the declaration to make it easier to read. There's not a lot there other than avoiding creating these things in memory unless we need them 🤷  removing one will definitely not have any performance impact but it's not a bad idea to have that on the back of our minds for when it matters I guess 😄